src/pygeosx/pygeosx.cpp

```diff
@@ -347,11 +360,15 @@ PyInit_pygeosx()
 {
   import_array();
 
-  LvArray::python::PyObjectRef<> module{ PyModule_Create( &pygeosxModuleFunctions ) };
-  if( module == nullptr )
-  {
-    return nullptr;
-  }
+  PyObject * module = PyModule_Create( &pygeosxModuleFunctions );
+  PyObject * submodule1 = geosx::python::PyInit_pysolver();
+  PyObject * submodule2 = geosx::python::PyInit_pyhdf5();
+
+  Py_INCREF( submodule1 );
+  PyModule_AddObject( module, "pysolver", submodule1 );
+
+  Py_INCREF( submodule2 );
+  PyModule_AddObject( module, "pyhdf5", submodule2 );
```

Comment:
Any particular reason these are added as new sub-modules?

Reply:
The idea is that pygeosx is the module to the whole "GEOSX machine" (especially initialization part of the machine), pysolver and pyhdf5 (and possibly other features) are components of this machine that have their own utilities. I don't know if I'm clear, but this is why I structured the things this way.

Reply:
Cool, just curious.